R/methods-Nodes.R

```diff
@@ -42,6 +42,14 @@ setMethod("getWeights", "NodeList", function(object) unlist(lapply(as.list(objec
 setMethod("getColors", "NodeList", function(object) unlist(lapply(as.list(object), color)))
 setGeneric("getDegrees", function(object) standardGeneric("getDegrees"))
 setMethod("getDegrees", "NodeList", function(object) unlist(lapply(as.list(object), degree)))
+setGeneric("getCoords", function(object) standardGeneric("getCoords"))
+setMethod("getCoords", "NodeList", function(object) {
+    coords <- data.frame(x = unlist(lapply(as.list(object), x)), y = unlist(lapply(as.list(object), y)))
```

Comment:
is it weird that the coords would come out with no nodeIds? I go back and forth

Reply:
yea. idk. igraph did it that way, but thats not necessarily a reason we should.. ill think about it.

Reply:
do getColors and getDegrees return named lists? i guess thats the real question.. we should be consistent.

Reply:
those dont appear to be named in the result, so im going to leave it. if we decide we dont like it someday, we can change them all at once as a separate issue.